docker/teleport-quickstart.yml

```diff
@@ -28,3 +28,20 @@ services:
       - ./teleport/data:/var/lib/teleport
     depends_on:
       - configure
+
+  #
+  # one-sshd is a single-node Teleport cluster called "one" (runs all 3 roles: proxy, auth and node)
+  #
+  one-sshd:
+    image: teleport:latest
+    container_name: one-sshd
+    command: /usr/bin/start-sshd.sh
+    env_file: env.file
+    mem_limit: 300m
+    volumes:
+      - ./sshd/pam.d/ssh:/etc/pam.d/ssh
+      - ./sshd/etc/ssh/sshd_config:/etc/ssh/sshd_config
```

Comment:
Making this change will prevent the `curl teleport-quickstart.yml && docker-compose -f teleport-quickstart.yaml up` flow that we've encouraged historically

Reply:
I want to phase out the old quickstart since 7.0, so I have removed the reference from it from the docs